scalameta/parsers/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3106,10 +3107,12 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
    */
   def traitDef(mods: List[Mod]): Defn.Trait = atPos(mods, auto) {
     val traitPos = in.tokenPos
-    val abstractMod = atPos(traitPos, traitPos)(Mod.Abstract())
+    val implicitMod = atPos(traitPos, traitPos)(Mod.Abstract())
```

Comment:
Implicit in the sense that `abstract` is assumed for trait modifiers. Will rename.